### DISCUSSION TITLE
need help with qemu 7.0.0

### DISCUSSION THREAD

**Discussion by CryptoMinerAndriod** [termux/termux-packages#10812] [Q&A]
which qemu-7.0.0 package has this https://github.com/qemu/qemu/blob/master/target/i386/cpu.c for a file? because I want to edit cpu.c to change CPU models

**Comment by ghost** [comment]
In the source code root directory `/target/i386/cpu.c`

**Reply by CryptoMinerAndriod** [reply]
so what folder in $PREFIX aka /data/data/com.termux/files/usr/ should I add it in?
